showcase/src/app/shared/reference-block/_reference-block.component.scss

```diff
@@ -0,0 +1,36 @@
+/* ============================================================================== */
+/*  S t a r k   R e f e r e n c e   B l o c k                                     */
+/* ============================================================================== */
+/* stark-ui: src/modules/reference-block/components/reference-block.component.scss */
+.reference-block {
+  background-color: #fff;
+  border: solid 1px color(mat-color($grey-palette, 300));
+  padding-left: 10px;
+  padding-top: 10px;
+  border-radius: 5px;
+  box-shadow: color(mat-color($grey-palette, 200));
+  position: relative;
+  margin-bottom: 15px;
+
+  & mat-icon {
+    display: inline-flex;
+    margin-right: 15px;
+  }
+  & ul {
+    margin-top: 5px;
+  }
+
+  & a {
+    text-decoration: none;
+  }
+  & a:hover {
+    background-color: $offwhite;
+    color: $md-primary;
+  }
+}
+.reference-title {
+  display: inline-flex;
```

Comment:
As we discussed, I've re-worked these styles to try to keep the same CSS as before, so this is what I got:

```css
.stark-reference-block div {
  background-color: #fff;
  border: solid 1px color(mat-color($grey-palette, 400));
  padding: 10px 10px 10px 50px;
  border-radius: 5px;
  box-shadow: color(mat-color($grey-palette, 300));
  position: relative;

  & mat-icon {
    position: absolute;
    left: 15px;
    top: 14px;
    color: color(mat-color($grey-palette, 700));
  }
  & ul {
    padding: 0;
  }

  & a {
    text-decoration: none;
  }
  & a:hover {
    background-color: $offwhite;
    color: $md-primary;
  }
}
```